### PR TITLE
Update Intel driver version for ansible deployment to the latest version.

### PR DESCRIPTION
**Pull request by johnkord**

### CODE REVIEW

File: scripts/ansible/roles/linux/intel/tasks/sgx-driver.yml [hunk 1]
```diff
@@ -2,13 +2,36 @@
...
+    install_recommends: no
+
+- name: Ensure aesmd service stopped
+  service:
+    name: aesmd
+    state: stopped
+
+- name: Download Intel SGX DCAP Driver
+  get_url:
+    url: https://download.01.org/intel-sgx/dcap-1.0.1/dcap_installer/ubuntuServer1604/sgx_linux_x64_driver_dcap_4f32b98.bin
```
[the remark this answers is not in the document]
Comment (ionutbalutoiu) [reply scripts/ansible/roles/linux/intel/tasks/sgx-driver.yml]: I agree with @vlaza.
It's a good practice to have this as a parameter into the [inventory/group_vars/linux-agents](https://github.com/Microsoft/openenclave/blob/87312cd05041343142fa48858824a36e9e67a5e2/scripts/ansible/inventory/group_vars/linux-agents) file.

[the remark this answers is not in the document]
Comment (johnkord) [reply scripts/ansible/roles/linux/intel/tasks/sgx-driver.yml]: Good idea, thanks all! I'll add this.

[the remark this answers is not in the document]
Comment (johnkord) [reply scripts/ansible/roles/linux/intel/tasks/sgx-driver.yml]: @vlaza @ionutbalutoiu When I try to deploy to localhost, I don't think it uses this linux-agents file.  I get this error when I try to reference the new variable I created:

```
TASK [linux/intel : Download Intel SGX DCAP Driver] *******************************************************************************************************************************************
fatal: [localhost]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: 'intel_sgx_driver_1604' is undefined\n\nThe error appears to have been in '/home/johnkord/tmp1/openenclave/scripts/ansible/roles/linux/intel/tasks/sgx-driver.yml': line 18, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: Download Intel SGX DCAP Driver\n  ^ here\n"}

```

[the remark this answers is not in the document]
Comment (vlaza) [reply scripts/ansible/roles/linux/intel/tasks/sgx-driver.yml]: @johnkord you can run the ansible playbook also with vars ar a workaround for `localhost` like this:

*-e VARS, --extra-vars=VARS
Extra variables to inject into a playbook, in key=value key=value format.*

**Pull request merged**